src/awscli_login/config.py

```diff
@@ -105,17 +105,17 @@ class Profile:
 
     def _init_dir(self) -> None:
         """ Create ~/.aws-login directory if it does not exist. """
-        home = expanduser('~')
+        root = environ.get('AWSCLI_LOGIN_ROOT')
 
-        self.home = home
+        self.home = root if root is not None else expanduser('~')
```

Comment:
If `AWSCLI_LOGIN_ROOT` is an empty string, `self.home` will be set to that value, causing an empty string to later be concatenated with various directories in the code at lines 113-118. Is this really what you want?

Reply:
```suggestion
self.home = root if root else expanduser('~')
```

Good catch Jon.

Reply:
Yes, I do. It allows users to use relative paths. It's an edge case I would not encourage but somebody may find it useful. Plus I'm not really doing any error checking to verify that AWSCLI_LOGIN_ROOT is an actual valid path to a directory. That would be the solution. So in short, an empty string is as valid as anything else from the point of view of this portion of the code.